src/modules/CRUD/components/DataTable/Header/index.js

```diff
@@ -20,5 +21,6 @@ export default compose(
   withRouter,
   connectAuthProvider(authProviderGetter),
   connectCRUDProvider('featuresList'),
+  connectUserSettingsProvider('tableSize', 'setTableSize'),
```

Comment:
This could be replaced by a simple function:

```suggestion
  withTableSize(),
```

*not mandatory.*